.github/workflows/rustdoc.yml

```diff
@@ -2,8 +2,12 @@ name: rustdoc
 
 on:
   push:
-   branches:
-   - master
+    branches:
+      - master
+    paths-ignore:
+      - '.github/**'
```

Comment:
I suggest including this one, since when workflows change, you may want to run these tests. Not a blocker, I'll still approve :)